proposals/20200224-combined-logging.md

```diff
@@ -0,0 +1,72 @@
+# Combined Logging
+
+Support the ability to view all of the container logs for a given workload. Currently you must request each container
+log separately.
+
+## Goals
+ - Combine the logging output from multiple containers.
+ - Tag/mark which log lines are from which container.
+ - Provide a list of which container logs are included in the stream.
+ - Only send content when there are new log entries.
+
+## Non-Goals
```

Comment:
Another suggestion I want to offer is that is combined logging for a pod. Multiple pods are not in scope.

Reply:
I'm thinking about this some more. I'm going to consider what complexity is added if we say this is to get logs combined from any workload vs. for all containers of a pod. I'm also going to consider how we might do this for only pods, but in way that makes it extensible for the next improvement, which would Application logs.